app/views/images/_index.html.erb

```diff
@@ -94,6 +101,41 @@
           <% end %>
         </td>
       </tr>
+      <td colspan="3">
+        <div class="dropdown" style="margin-top: -40px;">
+          <div class="dropdown-menu image-version" aria-labelledby="dropdownMenuButton" id="image-vesrions-<%= warpable.id %>">
+            <% warpable.versions.reverse.each do |version| %>
+              <button class="<%= warpable.id %>" id="<%= version.index + 1%>" onClick="getVersion(this)"><%= version.event.capitalize %>d by
+                <% if version.whodunnit && version.whodunnit != 'Anonymous' %>
+                  <% user = User.find_by_id(version.whodunnit) %>
+                  <% if user %>
+                    <a href="/profile/<%= user.login %>"><%= user.login %></a>
+                  <% end %>
+                <% else %>
+                  <u>an Anonymous user</u>
+                <% end %>
+                <%= time_ago_in_words(version.created_at) %> ago
+                <%= link_to '' ,images_revert_path(id: warpable.id, version:version), class: 'fa fa-undo', title: 'Revert to this version' %>
+              </button>
+            <% end %>
+          </div>
+        </div>
+      </td>
     <% end %>
   </tbody>
 </table>
+<script>
```

Comment:
Hi @divyabaid16, I broke this script when I change the functionality of `revert`...could you update(sorry my js sucks) it please or maybe we can remove it? Thanks